src/delta_lake/delta_service.py

```diff
@@ -33,6 +37,68 @@
     "vacuum",
 }
 
+DISALLOW_SQL_META_CHARS = {
+    "--",
+    "/*",
+    "*/",
+    ";",
+    "\\",
+}
+
+ALLOWED_STATEMENTS = {
+    "select",
+}
+
+FORBIDDEN_POSTGRESQL_SCHEMAS = {
+    # NOTE: This might create false positives, legitemate queries might include these schemas
+    # e.g. "SELECT * FROM jpg_files"
+    "pg_",
```

Comment:
```suggestion
FORBIDDEN_POSTGRESQL_SCHEMAS = {
    # NOTE: This might create false positives, legitemate queries might include these schemas
    # e.g. "SELECT * FROM jpg_files"
    # NOTE: may also need to expand this if other databases are used
    "pg_",
```